vardbg/output/video_writer/config.py

```diff
@@ -58,18 +61,27 @@ def parse_hex_color(string):
 
     return r, g, b, 255
 
+# Some color_scheme can have same txt and bg color
+def color_contrast(body_txt):
+    contrast = (int(body_txt[0] * 299) + int(body_txt[1] * 587) + int(body_txt[2] * 114)) / 1000
+    if (contrast >= 128):  
+        return (50,50,50,255) # return grey
+    else:
+        return (255,255,255,255) # return white
+
+# Calculate distance btw two color
+def color_similarity(base_col_val,oth_col_val):
+    return math.sqrt(sum((base_col_val[i]-oth_col_val[i])**2 for i in range(3)))
```

Comment:
This is no longer necessary.

Reply:
Ya, You are right, Sorry I forgot to remove this part from the code